src/main/java/bookface/model/person/Person.java

```diff
@@ -19,20 +20,34 @@ public class Person {
     private final Phone phone;
     private final Email email;
 
+    private final HashSet<Title> loanedBook = new HashSet<>();
```

Comment:
Can this be renamed to `loanedBooks`? As it is a `Collection`

Reply:
Done!